charts/kube-otel-stack/templates/collector.yaml

```diff
@@ -105,16 +105,16 @@ spec:
             scrape_interval: 5s
             static_configs:
             - labels:
-                collector_name: ${KUBE_POD_NAME}
+                collector_name: ${OTEL_K8S_POD_NAME}
               targets:
-                - 0.0.0.0:8888
+                - ${OTEL_K8S_POD_IP}:8888
```

Comment:
is there a reason to not just use 127.0.0.1 here?

Reply:
By Prometheus [conventions](https://www.robustperception.io/controlling-the-instance-label/) shouldn't the instance label uniquely identify a target?

Reply:
I believe that's a recommendation and the post is recommending to use some relabel configs (post-scrape) to do this. Regardless, I'm just trying to consider any risks by changing this to an IP address that isnt the local container. I believe this is probably a fine change to make, but was wondering if there was a reason this _had_ to change. Before merging, I'm going to test this on a local cluster to just ensure this will work as expected upon merge.

Reply:
As another example, the opentelemetry collector helm chart [also uses the pod IP](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/values.yaml#L122-L126) to self scrape.